pkg/cloudclient/gcp/gcp_test.go

```diff
@@ -1,16 +1,22 @@
 package gcp
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/openshift/cloud-ingress-operator/config"
 	"github.com/openshift/cloud-ingress-operator/pkg/testutils"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/googleapi"
+	"gotest.tools/assert"
 	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-func TestNewClient(t *testing.T) {
+var cli *Client
+
+func setupSuite(t *testing.T) {
```

Comment:
This is currently not being called anywhere.

Reply:
Yes, the setup and tearsdown functions are implicitly called, which lint doesn't seem to consider. We'll need to make a lint exception here. Not sure how that's done, will have to find out.

Reply:
Ref: https://coreos.slack.com/archives/CMD1L32M9/p1649693572235999?thread_ts=1649685655.318599&cid=CMD1L32M9

Reply:
Why was the name of the function changed?

Reply:
Before, creating the client was the test. Now creating the client is a part of set up so other tests in that suite can use it globally. I couldn't kept the old test, but it would be redundant if the setup was to do the same thing.

Reply:
I'm not following how setupSuite is implicitly called then.

Reply:
I thought it was, but it wasn't. I went down the rabbit hole a bit to make it call setupSuite implicitly, but that would need me to change the test to use the testify Suite struct, which is more than we want to do here. 

Reverted this to as before, moved the test to private_go.